.circleci/config.yml

```diff
@@ -382,65 +394,15 @@ jobs:
       - app-build-and-store:
           study_key: << pipeline.parameters.study_key >>
 
-  osteo-deploy-only:
-    executor:
-      name: commit-executor
-    steps:
-      - deploy-stored-build: &osteo-params
-          study_key: osteo
-          study_guid: CMI-OSTEO
-
-  brain-deploy-only:
-    executor:
-      name: commit-executor
-    steps:
-      - deploy-stored-build: &brain-params
-          study_key: brain
-          study_guid: cmi-brain
-
-  angio-deploy-only:
-    executor:
-      name: commit-executor
-    steps:
-      - deploy-stored-build: &angio-params
-          study_key: angio
-          study_guid: ANGIO
-
-  mbc-deploy-only:
-    executor:
-      name: commit-executor
-    steps:
-      - deploy-stored-build: &mbc-params
-          study_key: mbc
-          study_guid: cmi-mbc
-
-  osteo-build-and-deploy:
-    executor:
-      name: commit-executor
-    steps:
-      - app-build-and-deploy:
-          <<: *osteo-params
-
-  angio-build-and-deploy:
-    executor:
-      name: commit-executor
-    steps:
-      - app-build-and-deploy:
-          <<: *angio-params
-
-  brain-build-and-deploy:
-    executor:
-      name: commit-executor
-    steps:
-      - app-build-and-deploy:
-          <<: *brain-params
-
-  mbc-build-and-deploy:
```

Comment:
After refactor the specialized jobs that only differ by what key and guid is used can be removed.